<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>604</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>604</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VLF Handler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(208, 208, 208) </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayoutForGraphics">
         <item row="0" column="0">
          <widget class="QColumnView" name="columnView">
           <property name="styleSheet">
            <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionConcatenate"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Обработка</string>
    </property>
    <addaction name="actionCorrelation"/>
   </widget>
   <widget class="QMenu" name="menuGraphics">
    <property name="title">
     <string>Графики</string>
    </property>
    <widget class="QMenu" name="menu2D">
     <property name="title">
      <string>Двумерная визуализация</string>
     </property>
     <addaction name="action2DMake"/>
     <addaction name="action2DMakeWithMean"/>
    </widget>
    <widget class="QMenu" name="menu3D">
     <property name="title">
      <string>Трехмерная визуализация</string>
     </property>
     <addaction name="action3DMake"/>
     <addaction name="action3DRestore"/>
    </widget>
    <widget class="QMenu" name="menuSpectrum">
     <property name="title">
      <string>Спектральный график</string>
     </property>
     <addaction name="actionSpectrumMake"/>
     <addaction name="actionSpectrumRestore"/>
    </widget>
    <addaction name="menu2D"/>
    <addaction name="menu3D"/>
    <addaction name="menuSpectrum"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionShowHideDockInformation"/>
    <addaction name="actionShowHideDockInput"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuGraphics"/>
   <addaction name="menuView"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetInput">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>277</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>288</height>
    </size>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowIcon">
    <iconset>
     <normaloff>:/icons/rectangle.png</normaloff>:/icons/rectangle.png</iconset>
   </property>
   <property name="windowTitle">
    <string>Ввод</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0" columnstretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="3" column="0">
        <widget class="QLabel" name="labelMonth">
         <property name="text">
          <string>Месяц:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditMonth">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelDay">
         <property name="text">
          <string>День:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelYear">
         <property name="text">
          <string>Год:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelZenith">
         <property name="text">
          <string>Зенит:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditYear">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditLatitude">
         <property name="font">
          <font>
           <stylestrategy>NoAntialias</stylestrategy>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelLatitude">
         <property name="text">
          <string>Широта:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelLongitude">
         <property name="text">
          <string>Долгота:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEditZenith">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditDay">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditOffset">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelOffset">
         <property name="text">
          <string>Офсет:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditLongitude">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>9</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
    <zorder></zorder>
    <zorder>verticalSpacer</zorder>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetInformation">
   <property name="windowIcon">
    <iconset>
     <normaloff>:/icons/triangle.png</normaloff>:/icons/triangle.png</iconset>
   </property>
   <property name="windowTitle">
    <string>Информация</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>350</height>
        </size>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionCorrelation">
   <property name="text">
    <string>Корреляция</string>
   </property>
  </action>
  <action name="action2DMake">
   <property name="text">
    <string>Простая визуализация данных</string>
   </property>
  </action>
  <action name="action3DMake">
   <property name="text">
    <string>Простая визуалиация данных</string>
   </property>
  </action>
  <action name="action3DRestore">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Восстановить график</string>
   </property>
  </action>
  <action name="actionSpectrumMake">
   <property name="text">
    <string>Построить</string>
   </property>
  </action>
  <action name="actionSpectrumRestore">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Восстановить график</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Селекция</string>
   </property>
  </action>
  <action name="actionConcatenate">
   <property name="text">
    <string>Слияние</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionShowHideDockInformation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Панель информации</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionShowHideDockInput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Панель ввода</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMeans">
   <property name="text">
    <string>Среднее</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Только день</string>
   </property>
  </action>
  <action name="action2DMakeWithMean">
   <property name="text">
    <string>Среднее</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionSaveAsPic">
   <property name="text">
    <string>Сохранить рисунок...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditLatitude</tabstop>
  <tabstop>lineEditLongitude</tabstop>
  <tabstop>lineEditDay</tabstop>
  <tabstop>lineEditMonth</tabstop>
  <tabstop>lineEditYear</tabstop>
  <tabstop>lineEditZenith</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>lineEditOffset</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowHideDockInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidgetInput</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowHideDockInformation</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidgetInformation</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>766</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetInformation</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionShowHideDockInformation</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetInput</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionShowHideDockInput</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
